### DISCUSSION TITLE
[插件请求]隐藏视频结束后的推荐

### DISCUSSION THREAD

**Discussion by Skylarzz** [the1812/Bilibili-Evolved#3662] [功能建议 / Ideas]
![图片](https://user-images.githubusercontent.com/14090545/188327068-4b19fb73-578d-442d-b8dc-2d146e5abacb.png)

**Comment by the1812** [comment]
`隐藏视频推荐` 不就可以

**Reply by Skylarzz** [reply]
> `隐藏视频推荐` 不就可以

播放器里面的还在，播放结束后就粗现

![图片](https://user-images.githubusercontent.com/14090545/189943970-fd51796f-8a6d-45d4-b62c-6c4a43c866ae.png)

**Reply by Geeyun-JY3** [reply]
话说这功能最初到底能否隐藏播放结束后播放器里的推荐视频。
功能描述里也没明确指出：https://github.com/the1812/Bilibili-Evolved/blob/master/doc/features/features.md#%E9%9A%90%E8%97%8F%E8%A7%86%E9%A2%91%E6%8E%A8%E8%8D%90
